packages/data/json/terms.json

```diff
@@ -7,33 +7,33 @@
     "definition": "<glossary-term>Alteration</glossary-term> means a change or extension to any matter or thing or to any <defined-term-glossary>occupancy</defined-term-glossary> regulated by this Code.",
     "cleanDefinition": "Alteration means a change or extension to any matter or thing or to any occupancy regulated by this Code."
   },
-  "appliances": {
-    "definition": "<glossary-term>Appliances</glossary-term> means a device to convert fuel into energy and includes all components, controls, wiring and piping required to be part of the device by the applicable standard referred to in this Code.",
+  "appliance": {
+    "definition": "<glossary-term>Appliance</glossary-term> means a device to convert fuel into energy and includes all components, controls, wiring and piping required to be part of the device by the applicable standard referred to in this Code.",
     "cleanDefinition": "Appliances means a device to convert fuel into energy and includes all components, controls, wiring and piping required to be part of the device by the applicable standard referred to in this Code."
   },
-  "assembly occupancies": {
-    "definition": "<glossary-term>Assembly Occupancies</glossary-term> means the <defined-term-glossary>occupancy</defined-term-glossary> or the use of a <defined-term-glossary>building</defined-term-glossary> or part thereof by a gathering of persons for civic, political, travel, religious, social, educational, recreational or like purposes, or for the consumption of food or drink.",
-    "cleanDefinition": "Assembly Occupancies means the occupancy or the use of a building or part thereof by a gathering of persons for civic, political, travel, religious, social, educational, recreational or like purposes, or for the consumption of food or drink."
+  "assembly occupancy": {
+    "definition": "<glossary-term>Assembly occupancy</glossary-term> means the <defined-term-glossary>occupancy</defined-term-glossary> or the use of a <defined-term-glossary>building</defined-term-glossary> or part thereof by a gathering of persons for civic, political, travel, religious, social, educational, recreational or like purposes, or for the consumption of food or drink.",
```

Comment:
I think the comment in the ticket is mean that Group A is A1-A4. Can we do something like:

```
<defined-term-glossary override-term='major occupancy' override-tooltip='Groups A1 through A4'>(Group A)</defined-term-glossary>
```